CHANGES.rst

```diff
@@ -174,6 +174,11 @@ ramp_fitting
 
 - Moving some ramp fitting CI tests from JWST to STCAL. [#8060]
 
+ramp fit
+-------
```

Comment:
The underline needs one 1 more character, to match the length of the line above.

Reply:
This whole entry needs to be moved to the top of the change log, under the unreleased section.